project/plugins.sbt

```diff
@@ -16,6 +16,6 @@ addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.8.0")
 // The following plugin is otherwise known as neo-sbt-scalafmt
 // - see: https://github.com/lucidsoftware/neo-sbt-scalafmt
 // TODO: CO-77: load scalafmt plugin
-// addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.1")
+addSbtPlugin("com.lucidchart" % "sbt-scalafmt-coursier" % "1.7")
```

Comment:
As we are using coursier, it is now the correct thing to do. https://github.com/lucidsoftware/neo-sbt-scalafmt/blob/master/README.md#usage points out the importance of doing this in this type of use case.